apps/avatax/src/modules/avatax/calculate-taxes/avatax-calculate-taxes-response-shipping-transformer.ts

```diff
@@ -62,6 +62,14 @@ export function transformAvataxTransactionModelIntoShipping(
     shippingLine.details?.map((details) => details.rate),
   );
 
+  const hasFee = shippingLine.details?.some((details) => details.isFee);
+
+  if (hasFee) {
+    logger.warn("Shipping line has a fee. App will report this fee as shipping_tax_rate", {
```

Comment:
I'm open to change but we are wrongly reporting rate to Saleor - maybe for Colorado Tax it will be fine (as this is different product - at least if client uses our recommended integration guide) but for other cases we may not be lucky 🤔

Reply:
If this is wrong, maybe this should be error?